### PR TITLE
Add Dutch translation

### PR DESCRIPTION
**Pull request by d9h02f**

### CODE REVIEW

File: _includes/language.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 	<li><div>Switch Language</div></li>
 	<li><a href="{{site.baseurl}}/">English</a></li>
 	<li><a href="{{site.baseurl}}/fr/">Français</a></li>
+	<li><a href="{{site.baseurl}}/nl/">Nederlands</a></li>
```
Comment (wildskyf) [inline _includes/language.html]: @d9h02f yo, does Nederland has a 's' ?

File: _config.yml [hunk 2]
```diff
@@ -1,11 +1,13 @@
 baseurl: "/lollypop-web"
-version: 0.9.98
+version: 0.9.112
```
Comment (wildskyf) [inline _config.yml]: @d9h02f why is the version 112?
(I am just curious)

**Comment by wildskyf** [comment]
I know nothing about Dutch, so after you answer/fix the comment I left, this PR might be merge!

Ping @gnumdk , @bil-elmoussaoui , anything else?

File: _includes/language.html (hunk 1 above)
Comment (d9h02f) [inline _includes/language.html]: There's a country called Nederland but the language is Nederlands.

File: _config.yml (hunk 2 above)
Comment (d9h02f) [inline _config.yml]: I noticed @gnumdk upped the version in this commit https://github.com/gnumdk/lollypop-web/commit/096dabe7d042ecaa71531dd61c4684adfed97a36 and assumed it was the latest Lollypop version. I don't know if it even changes anything.

File: _includes/language.html (hunk 1 above)
Comment (wildskyf) [inline _includes/language.html]: Got it !

Comment (bil-elmoussaoui) [inline _includes/language.html]: Will check this and report back tomorrow!

**Comment by d9h02f** [comment]
bump

**Comment by wildskyf** [comment]
Thanks a lot! Sorry for forgetting your PR.

**Comment by d9h02f** [comment]
No problem, thanks for merging.

**Pull request merged**